rosa_hcp/rosa-hcp-creating-cluster-with-aws-kms-key.adoc

```diff
@@ -0,0 +1,82 @@
+:_content-type: ASSEMBLY
+[id="rosa-hcp-creating-cluster-with-aws-kms-key"]
+= Creating an AWS KMS key for customer encryption
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: rosa-hcp-creating-cluster-with-aws-kms-key
+
+toc::[]
+
+Create a {product-title} (ROSA) cluster with the AWS Security Token Service (STS) using a custom AWS Key Management Service (KMS) key.
```

Comment:
We should use `{hcp-title-first}` here https://github.com/openshift/openshift-docs/blob/main/_attributes/attributes-openshift-dedicated.adoc#L38